treadmill/infra/__init__.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+SCRIPT_DIR = os.path.join(
```

Comment:
please use pkg_util for all resource discovery - see bootstrap/__init__.py for examples.